hoomd/md/pytest/conftest.py

```diff
@@ -0,0 +1,533 @@
+import hoomd
+import pytest
+import numpy as np
+from copy import deepcopy
+import itertools
+
+np.random.seed(0)
+
+
+def _make_valid_param_dicts(arg_dict, n_dicts):
+    # turn {'a': [0, 1], 'b':[2, 3]} into [{'a': 0, 'b': 2}, {'a': 1, 'b': 3}]
+    return [dict(zip(arg_dict, val)) for val in zip(*arg_dict.values())]
+
+
+def _valid_params(particle_types=['A', 'B']):
```

Comment:
All of these test fixtures belong in `test_pair.py` as they are specific to those tests and not all MD tests in general.

Reply:
Resolved 92252586f